provisioning/tasks/install-site.yml

```diff
@@ -14,7 +14,7 @@
     --site-name="{{ drupal_site_name }}"
     --account-name={{ drupal_account_name }}
     --account-pass={{ drupal_account_pass }}
-    --db-url=mysql://{{ drupal_mysql_user }}:{{ drupal_mysql_password }}@localhost/{{ mysql_databases[0].name }}
+    --db-url={{ drupalvm_database }}://{{ drupal_mysql_user }}:{{ drupal_mysql_password }}@localhost/{{ drupal_mysql_database }}
```

Comment:
Wouldn't the latter variable need to work with either mysql or pgsql? Or I guess that variable could just change full stop to `drupal_database_name`, perhaps?